tests/test_introduction_to_classes.py

```diff
@@ -1,14 +1,14 @@
 import pytest
 
 from codecademy.introduction_to_classes import Animal
+from codecademy.introduction_to_classes import Animal6
 from codecademy.introduction_to_classes import Customer
+from codecademy.introduction_to_classes import Employee14, PartTimeEmployee14
 from codecademy.introduction_to_classes import Fruit
 from codecademy.introduction_to_classes import ReturningCustomer
-from codecademy.introduction_to_classes import Triangle
-from codecademy.introduction_to_classes import Animal6
-from codecademy.introduction_to_classes import ShoppingCart10
 from codecademy.introduction_to_classes import Shape12, Triangle12, Equilateral12
-from codecademy.introduction_to_classes import Employee14, PartTimeEmployee14
+from codecademy.introduction_to_classes import ShoppingCart10
+from codecademy.introduction_to_classes import Triangle
```

Comment:
@nicolas13sochi chain imports is this:
```python
from codecademy.introduction_to_classes import Animal, Animal6, Customer, Employee14, PartTimeEmployee14, Fruit, ReturningCustomer, Shape12, Triangle12, Equilateral12, ShoppingCart10, Triangle
```
But you need to sort them alphabetically